sortiment/users/forms.py

```diff
@@ -31,6 +31,7 @@ class CreditMovementForm(forms.Form):
         .filter(is_guest=False),
         label="Používateľ",
     )
+    message = CharField(max_length=1000, label="Správa")
```

Comment:
Nejaký rozumnejší max_length?

```suggestion
    message = CharField(max_length=128, label="Správa")
```

Potom treba pregenerovať migráciu – najjednoduchšie ju vymazať a spraviť nanovo.